.github/workflows/default.yml

```diff
@@ -13,10 +13,16 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: files
+        name: Get changed files
+        uses: jitterbit/get-changed-files@v1
```

Comment:
I'd avoid bringing an external Action for something that's easily solved with git.

Reply:
I was thinking the same but I don't think it is easily solved with the information the git workflows give you. Look at the implementation of some of these things:

https://github.com/tj-actions/changed-files/blob/main/entrypoint.sh
https://github.com/lots0logs/gh-action-get-changed-files/blob/master/index.js
https://github.com/jitterbit/get-changed-files/blob/master/src/main.ts

Reply:
We could also always change it later. Doing this for now doesn't block us from making changes in the future etc.

This thing seems well used and we version lock it so I think it's stable for now. Unless we can figure it out with a 1-2 line git command I don't want to waste anymore time on this.